lib/routes/index.js

```diff
@@ -11,16 +11,21 @@ module.exports = function (
   serverPublicKeys,
   signer,
   db,
+  oauthdb,
   mailer,
   smsImpl,
   Password,
   config,
   customs
   ) {
+  // Various extra helpers.
+  const push = require('../push')(log, db, config)
+  const pushbox = require('../pushbox')(log, config)
+  const devicesImpl = require('../devices')(log, db, push)
+  const signinUtils = require('./utils/signin')(log, config, customs, db, mailer)
```

Comment:
I just moved these around, they were all intermingled with the actual route imports.